rskj-core/src/main/java/co/rsk/NodeBootstrapper.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk;
+
+import co.rsk.cli.CliArgs;
+import co.rsk.config.NodeCliFlags;
+import co.rsk.config.NodeCliOptions;
+import org.ethereum.config.DefaultConfig;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+/**
+ * Bootstraps the node, creating the initial object graph.
+ * It uses Spring at the moment.
+ */
+class NodeBootstrapper {
```

Comment:
I think it's better to use `public`/`private` explicitly instead of the _default_ visibility